t/fmt-asciidoc.t

```diff
@@ -55,6 +55,11 @@ push @tests,
     'format'  => 'asciidoc',
     'options' => "-o yfm_skip_array -o yfm_keys='title , subtitle,paragraph'",
     'input'   => "fmt/asciidoc/YamlFrontMatter_Option.adoc",
+  },
+  {
+    'format'  => 'asciidoc',
+    'input'   => 'fmt/asciidoc/MacroIncludesHugo.adoc',
+    'doc'     => 'asciidoctor includes with hugo shortcode',
   };
```

Comment:
The `MacroIncludesHugo.adoc` should simply be included in the list on line 15 as it does not require a special command line option.

Reply:
Ok, I'll change that.